### PR TITLE
k8s 1.11.0 provider with multus network plugin

### PR DESCRIPTION
**Pull request by SchSeba**
**What this PR does / why we need it**:
k8s 1.11.0 provider with multus network plugin.
Needed to start working on multus integration.

**Release note**:

```release-note
Added k8s 1.11.0 provider with multus network plugin
```

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/kubevirt/1380)

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
@SchSeba We have https://github.com/kubevirt/kubevirt/pull/1353, that has the number of not flaky failures, so I prefer to wait until this one will be merged.

**Comment by SchSeba** [comment]
@cynepco3hahue No problem.

File: stdci.yaml [hunk 1]
```diff
@@ -2,6 +2,8 @@
 sub-stages:
   - k8s-1.10.3-dev
   - k8s-1.10.3-release
+  - k8s-multus-1.11.0-dev
+  - k8s-multus-1.11.0-release
```
Comment (rmohr) [inline stdci.yaml]: Just release should be good enough.

**Comment by cynepco3hahue** [comment]
@SchSeba Please rebase and fix Roman comment and we can continue with it.

**Comment by SchSeba** [comment]
@cynepco3hahue ok but before this multus make a big change in the api yesterday,  so I need to build the provider again in the kubevirtci project. I will make the change there and come back to this PR after that.

**Comment by phoracek** [comment]
@SchSeba we could deploy Multus following the new "Quick Start guide". I created a manifest that installs it on kubevirtci: https://raw.githubusercontent.com/phoracek/ovs-network-plugin/master/manifests/multus.yml

Hope it helps.

**Comment by SchSeba** [comment]
Close this PR.
This changes are in inside #1428

**Pull request closed without merging**